modules/nf-core/wipertools/fastqscatter/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+nextflow_process {
+
+    name "Test Process WIPERTOOLS_FASTQSCATTER"
+    script "../main.nf"
+    process "WIPERTOOLS_FASTQSCATTER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "wipertools"
+    tag "wipertools/fastqscatter"
+
+    test("3 reads, one truncated - fastq") {
+
+        config "./base.config"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/fastqrepair/testdata/truncated_clean.fastq', checkIfExists: true),
```

Comment:
can you correct the paths please? :)

Reply:
thank you. I don't understand how to correct the path. Can you advice?

Reply:
we usually give it like this:
`file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/vcf/chr21/simulated_sv.vcf.gz', checkIfExists: true)`
But i can see that you are using the test data from the fastqrepair branch (i guess?) -> so what you would do is open a PR to the modules branch of the test_data repo and add your data there :)

Reply:
I am wondering if we could leave it as is - I just have alwas seen that modules should take their test data from the `modules` branch

Reply:
I asked on slack lets see what people think - then I'll either approve the PR or we change the filepaths. Hope that is ok for you!